frontend/src/app/settings/modelsources/git-settings/edit-git-settings/edit-git-settings.component.ts

```diff
@@ -21,18 +21,19 @@ import {
 export class EditGitSettingsComponent implements OnInit, OnDestroy {
   id: number = -1;
 
-  gitSettingsForm = new FormGroup({
-    type: new FormControl('', Validators.required),
-    name: new FormControl('', Validators.required),
-    url: new FormControl('', Validators.required),
-    apiURL: new FormControl(''),
+  gitSettingsForm = this.fb.group({
+    type: this.fb.control('', Validators.required),
+    name: this.fb.control('', Validators.required),
+    url: this.fb.control('', Validators.required),
+    apiURL: this.fb.control(''),
```

Comment:
What's the difference between the `FormBuilder` and `FormGroup` / `FormControl`. I prefer the latter one, because no injectable is needed, so it's one line less and we use it everywhere else in the code.

Reply:
The first advantage of this approach is that we can decrease the imports at the top as it usually takes several lines to import the `FormGroup` and `FormControl` while by using the `FormBuilder` this can be reduced to just one single line.
An additional advantage of this approach is the reduction of duplicated/redundant code which this code snippet here does not yet reflect. If you look at the last commit you can see that by using the FormBuilder you can remove all the 'new FormControl(...)` or here the `this.fb.control(...)`. So for me using the FormBuilder (that also is just there to provide a convenient method to define forms) is better than directly using the 'FormGroup` and `FormControl`.